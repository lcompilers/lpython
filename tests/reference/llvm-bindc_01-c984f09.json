{
    "basename": "llvm-bindc_01-c984f09",
    "cmd": "lpython --no-color --show-llvm {infile} -o {outfile}",
    "infile": "tests/../integration_tests/bindc_01.py",
    "infile_hash": "3402ae1e5ed454e63ac3105da5d7264c880b666ac0ae672bf6b2dcfe",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "llvm-bindc_01-c984f09.stdout",
    "stdout_hash": "7e1b82f490b9a95ea81f4754694a725b89f16080d738e77e35462e9f",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}