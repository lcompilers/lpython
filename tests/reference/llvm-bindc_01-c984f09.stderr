code generation error: asr_to_llvm: module failed verification. Error:
Stored value type does not match pointer operand type!
  store i16** @x, i16* %_lpython_main_program1, align 8
 i16
